Automattic/wp-calypso
# New invite form: Redesign passwordless email invitation

Related to #86758

## Proposed Changes

* This is part of the passwordless invite form (pdDR7T-1m0-p2#comment-1564) . This PR implements the proposed design #86758
* It should be backwards compatible, meaning all the other features using passwordless components should remain the same, for instance you can think of login, create account, forgot password, etc.

## Testing Instructions



* Apply this branch or use live link
* Using your atomic site, invite a new user. Follow these https://github.com/Automattic/wp-calypso/pull/87083
* The email should look like the proposed design.
* Check also sign in, sign up, forgot password and login, it should and behave as before
<img width="2048" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/9ee55dda-a888-4e36-8fa3-ba4998aa56e6">

<img width="432" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/84b586e3-ff31-43e0-9333-e48f60dc1755">


## Pre-merge Checklist
